scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/qfn-20.yaml

```diff
@@ -542,6 +542,69 @@ QFN-20-1EP_4x5mm_P0.5mm_EP2.65x3.65mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+QFN-20-1EP_5x5mm_P0.65mm_EP3.1x3.1mm:
+  device_type: 'QFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://pdfserv.maximintegrated.com/package_dwgs/21-0140.PDF'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 5
+    tolerance: 0
+  body_size_y:
+    nominal: 5
+    tolerance: 0
+  overall_height:
+    minimum: 0.7
+    nominal: 0.75
+    maximum: 0.8
+
+  lead_width:
+    minimum: 0.25
+    nominal: 0.30
+    maximum: 0.35
+  lead_len:
+    minimum: 0.45
+    nominal: 0.55
+    maximum: 0.65
+
+  EP_size_x:
+    minimum: 3.0
+    nominal: 3.1
+    maximum: 3.2
+  EP_size_x_overwrite: 3.1
```

Comment:
Remove this as the nominal is the EP size you want. Same with Y dimension below.